mingw-w64-git/git-wrapper.c

```diff
@@ -164,13 +204,20 @@ static void setup_environment(LPWSTR top_level_path, int full_path)
 
 	/* extend the PATH */
 	len = GetEnvironmentVariable(L"PATH", NULL, 0);
-	len = sizeof(WCHAR) * (len + 3 * MAX_PATH);
+	len = sizeof(WCHAR) * (len + 4 * MAX_PATH);
 	path2 = (LPWSTR)malloc(len);
 	wcscpy(path2, top_level_path);
 	if (!full_path)
 		my_path_append(path2, L"cmd;", len);
 	else {
 		my_path_append(path2, msystem_bin, len);
+		if (IsRunningOnARM64()) {
+			/*
+			 * Many modules aren't available natively for ARM64 yet, but we can leverage i686 emulation.
+			 * Therefore we add /minw32/bin to the path.
+			 */
+			my_path_append(path2, L".../mingw32/bin", len);
```

Comment:
This "..." is not correct, of course... We need to do something like this instead:

```c
			wcscat(path2, L";");
			wcscat(path2, top_level_path);
			my_path_append(path2, L"mingw32\\bin;", len);
```